app/pb_kits/playbook/pb_image/_image.scss

```diff
@@ -1,15 +1,42 @@
-.pb_image_kit {
-	max-width: 100%;
-	max-height: 100%;
-
-	.blur_up {
-		-webkit-filter: blur(6px);
-		filter: blur(6px);
-		transition: filter 550ms, -webkit-filter 550ms;
-	}
-
-	.blur_up.lazyloaded {
-		-webkit-filter: blur(0);
-		filter: blur(0);
-	}
+@import "../tokens/border_radius";
+@import "../tokens/colors";
+@import "../pb_image/image";
+
+$image-sizes: (
+  "xs": 60px,
+  "sm": 80px,
+  "md": 100px,
+  "lg": 120px,
+  "xl": 140px,
+);
+
+[class^=pb_image_kit] {
+  position: relative;
+
+  @each $name, $size in $image-sizes {
+    &[class*=_#{$name}] {
+      width: $size;
+      height: $size;
+      object-fit: cover;
+      position: relative;
+      flex-shrink: 0;
+      flex-grow: 0;
+      flex-basis: $size;
+    }
+
+    &[class*= rounded] {
```

Comment:
This could be moved out of the loop so unsized images could have rounded corners

Reply:
We can pass the rounded prop with a default size of "none" and still achieve the desired result. If we remove this from the loop, rounded no long does what it is intended.

Reply:
i think whether its in the loop or not, it works regardless